.github/workflows/source-web-tool-scripts.yaml

```diff
@@ -0,0 +1,57 @@
+name: Source web tool scripts in container r2dii/r-packages
+on: push
+jobs:
+  Source-web-tool-scripts:
+    container: 2dii/r-packages:latest
+    runs-on: ubuntu-latest
+    name: Source web tool scripts in container r2dii/r-packages
+    strategy:
+      fail-fast: false
+    env:
+      R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      # When checking out the repository that triggered a workflow, `ref:`
+      # defaults to the reference or SHA for that event. Otherwise, uses the
+      # default branch --https://github.com/actions/checkout
```

Comment:
Here the repo that triggers the workflow is r2dii.climate.stress.test. So the workflow runs with the last commit on this PR and the default branch of all other repos. That means the same workflow is used with different intentions here and in PACTA_analysis. Here it helps us notice if any change here breaks the webtool. In PACTA_analysis it helps notice if any change in PACTA_analysis breaks the webtool -- and as a side effect it helps detect changes in the friends of PACTA_analysis that have breaking changes in the default branch. This of course would not happen if all the friends of PACTA_analysis did what we are doing here -- i.e. to avoid merging into the default branch any PR that breaks the web tool.